server/sonar-server/src/test/java/org/sonar/server/db/migrations/PlatformDatabaseMigrationAsynchronousTest.java

```diff
@@ -33,15 +33,15 @@
    * Implementation of execute wraps specified Runnable to add a delay of 200 ms before passing it
    * to a SingleThread executor to execute asynchronously.
    */
-  private PlatformDatabaseMigrationExecutorService executorService = new PlatformDatabaseMigrationExecutorServiceAdaptor() {
+  PlatformDatabaseMigrationExecutorService executorService = new PlatformDatabaseMigrationExecutorServiceAdaptor() {
     @Override
     public void execute(final Runnable command) {
       taskSuppliedForAsyncProcess = true;
     }
   };
-  private RubyBridge rubyBridge = mock(RubyBridge.class);
-  private Platform platform = mock(Platform.class);
-  private PlatformDatabaseMigration underTest = new PlatformDatabaseMigration(rubyBridge, executorService, platform);
+  RubyBridge rubyBridge = mock(RubyBridge.class);
```

Comment:
what's the point of changing this test classes just to remove some private keywords? (this is no debt)
Anyway, it's unrelated to this commit and shouldn't be here.